.gitmodules

```diff
@@ -1,6 +1,3 @@
-[submodule "extern/filecoin-ffi"]
-	path = extern/filecoin-ffi
-	url = https://github.com/filecoin-project/filecoin-ffi
 [submodule "extern/fil-blst"]
```

Comment:
commentary: I wonder if we can remove this too..

Reply:
this one i don't have strong feelings about because it doesn't have the extra build step to make the go interface available.